commit b95853bd4839e5f9f0aa6afee373f8b5a6d3be11 (refs/changes/74/474/2)
Author: David Pacheco <dap@joyent.com>
Date:   2016-09-16T10:16:53-07:00 (3 years, 1 month ago)
    
    TOOLS-1546 cr.joyent.us "new user guide" link overlaps with header on change screen
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/images/appserver/theme/GerritSite.css b/images/appserver/theme/GerritSite.css
index 63dfb71..6be053c 100644
--- a/images/appserver/theme/GerritSite.css
+++ b/images/appserver/theme/GerritSite.css
@@ -2,10 +2,10 @@
  * See the Gerrit documentation for information about this file.
  */
 table.crjHeader {
-	width: 100%;
 	position: fixed;
 	right: 5px;
-	top: 35px;
+	top: 27px;
+	width: 300px;
 
 	font-weight: bold;
 	text-align: right;
